en/includes/guides/authentication/conditional-auth/configure-conditional-auth.md

```diff
@@ -106,4 +106,14 @@ To delete an existing secret:
 
 5. Select the checkbox and confirm your action.
 
-{% endif %}
\ No newline at end of file
+### Using Secrets in Scripts
+
+You may refer to the added secrets in your conditional authentication scripts using the `secrets.key` syntax. For example, to retrieve a secret value, you may use:
```

Comment:
Adding secrets only talk about adding the secret name and the value. So here you should refer that the user can refer to the secret by using the secret name added in step x. The syntax should be secrets.{secret name}

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4773/commits/9acc22f1d66873e2c67cb71939862187674bece9